baseline/OSPS-BR.yaml

```
@@ -62,20 +62,19 @@ criteria:
   - id: OSPS-BR-03
     maturity_level: 1
     criterion: |
-      Any websites, API responses or other
-      services involved in the project development
+      Any websites and version control systems
+      involved in the project development
       and release MUST be delivered using SSH,
```

```suggestion
      MUST be delivered using SSH,
```

Did you intend to leave "release" in here? It seems like that part is better served in the two new criteria, but I might misunderstand the intent.

---

No, 53-character line folding screwed me up.